.github/ISSUE_TEMPLATE.md

```diff
@@ -1,16 +1,17 @@
+---
+
 ### Environment summary
 
 **Install Method:** How did you install the CLI? (e.g. pip, interactive script, apt-get, Docker, MSI, nightly)  
 Answer here:
 
-**CLI Version:** What version of the CLI and modules are installed? (Available with `az --version`)  
+**CLI Version:** What version of the CLI and modules are installed? (The full output of `az --version`)  
```

Comment:
I think what you are going for here is "Put the version of the MODULE you are using, not just azure-cli."  Maybe there's a way you could word it to encourage that without resulting in people always posting every module version (which is and will continue to be annoying since 95% of that is always irrelevant).